rust/src/page_table.rs

```diff
@@ -0,0 +1,57 @@
+use std::io::{Read, Seek, SeekFrom};
```

Comment:
Please add license once it is ready to merge.

Reply:
Doh! Is there a command line tool that can help add and check file headers, so we don't need to care about it in the review?